workflows/mag.nf

```diff
@@ -241,17 +241,8 @@ workflow MAG {
             ADAPTERREMOVAL_PE ( ch_adapterremoval_in.paired, [] )
             ADAPTERREMOVAL_SE ( ch_adapterremoval_in.single, [] )
 
-            // pair1 and 2 come for PE data from separate output channels, so bring
-            // this back together again here
-            ch_adapterremoval_pe_out = Channel.empty()
```

Comment:
This deletion may not work, as if you do merging of PE data you won't have the `truncated` output channel from the AR module (reviewed from my phone, so I'm not 💯 sure)

Reply:
Thanks, will check

Reply:
Actually ignore me, I confused myself - you don't want to do merging in this pipeline :sweat_smile: so it's irrelevent! You're correct , the output channel is just empty because the module wouldn't get executed.